talk/expert/cpp20concepts.tex

```diff
@@ -0,0 +1,194 @@
+\subsection[tmpl]{C++20 Requirements and Concepts}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{C++20 Requirements and Concepts}
+  \begin{block}{Motivation}
+    Generic programming is made of functions and class templates
+    which can be instantiated with different types. It is frequent
+    to instantiate them with {\em irrelevant types}, and the resulting
+    compilation errors are generally very long and hardly understandable.
+    As a last resort, the template authors are providing {\em documentation}
+    about the relevant parameters, and practice some tricky
+    {\em template meta-programmation}. C++20 finally brings simpler ways to
+    define constraint on template parameters ! Among different proposals,
+    the ISO committee has validated the flavor known as {\em Concepts Lite}.
```

Comment:
Is this relevant? Why would a student care which concept approach was finally taken?

Reply:
When I was talking about concepts 1-2 years ago, someone looking for concepts information on internet may face several flavors, amd may find useful to know that the validated ones are the "Concepts Lite". As you say, not sure if is useful today. And not sure a student will care about. Someone else opinion ?

Reply:
I remove.